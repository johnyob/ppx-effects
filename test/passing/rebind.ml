exception%effect E : string
exception%effect F = E
