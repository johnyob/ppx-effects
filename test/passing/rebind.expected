open! Ppx_effects_runtime.Import
type _ Ppx_effects_runtime.t +=  
  | E: string Ppx_effects_runtime.t 
type _ Ppx_effects_runtime.t +=  
  | F = E
